autoload/committia/git.vim

```diff
@@ -185,6 +201,29 @@ function! committia#git#status() abort
         " Leave status window empty when git-dir or work-tree not found
         return ''
     endtry
+    if g:committia#git#use_verbose || exists('s:use_verbose_status')
+        if exists('s:use_verbose_status')
+            unlet s:use_verbose_status
+        end
+        let scissors_line = search('^[#;@!$%^&|:] -\+ >8 -\+\n', 'cenW')
+        if scissors_line > 1
+            " Localisation hack, find the start of the status in the commit
+            " message template using the first line of output from `git status`
+            " Search backwards to avoid match in message, and start search at
+            " scissors line to avoid potential match in diff, unlikely, but...
+            let status_start = scissors_line
+            let re_status_start = '^[#;@!$%^&|:] ' . split(status, '\n')[0]
+            while status_start > 1
+                if match(getline(status_start - 1), re_status_start) != 0
```

Comment:
```suggestion
                if getline(status_start - 1) !~# re_status_start
```

`match()` changes its behavior looking at `'ignorecase'` option. It is a good idea to avoid it as much as possible.

Reply:
Thanks, will revisit, I have force pushed since that change btw as I spotted a bug, but I'll replace any use of `match()` in another commit :-)

Reply:
Fixed in 531ac91b, I may have gone overboard applying the change whether the pattern matches on alphabetical characters or not, but it seemed sensible to be consistent